text/0002-split-coinstake-transaction/0002-split-coinstake-transaction.md

```diff
@@ -8,25 +8,23 @@
+This RFC depends on [RFC-0004](../0004-remove-transaction-timestamp/0004-remove-transaction-timestamp.md) and enables [RFC-0003](../0003-multisig-minting/0003-multisig-minting.md)
 
 ## Conventions
 - The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
 - The verb "TO MINT" or "MINTING" refers to the action of trying to create a Block using Proof-of-Stake. "STAKING" is chosen over "MINTING" to avoid confusion with "MINING".
 
 ## Motivation
-Multiple community members have been asking about *cold minting* and the power of minters to add *free transactions* to their blocks.
-While all this time, a potential solution for both those problems has been sitting in every Proof-of-Stake block, the unused coinbase transaction.
-Although the splitting of the coinstake transaction has been described extensively by mquandalle [[1]](https://gist.github.com/mquandalle/7fe702a595f07f4b0f81), it has never been brought up as a possible alternative for *cold minting*. This RFC focuses on the discussion about splitting the coinstake transaction, the discussion about *multi-signature minting* is part of [RFC-0003](../0003-multisig-minting/0003-multisig-minting.md).
+The Peercoin community has sought for a solution to *cold minting* that does not severely complicate the process of adopting code from the Bitcoin ecosystem.
+While it is not entirely clear what features constitute 'cold minting', this proposal outlines a particularly protocol-sensitive path to achieve *air gapped minting*, whereby the key used at the time of minting is not sufficient to also move the coins.
 
-Minters are able to include transaction data without paying the fee when they create a block, limited only by the blocksize.
-As a minter of a small number of blocks might not hold many coin at stake, this can be a method by which a bad actor can drastically bloat the chain without much personal loss.
-When splitting the coinstake transaction, a natural limit of 1 free KB per minted block will provide minters with flexibility to adjust their outputs as desired while preventing spam.
+Although the splitting of the coinstake transaction has been described extensively by mquandalle [[1]](https://gist.github.com/mquandalle/7fe702a595f07f4b0f81), it has never been brought up as a possible alternative for *cold minting*.
+While the entire mechanisms relies on multiple RFCs, this RFC focuses on the discussion about splitting the coinstake transaction, as it is the most controversial part of the proposal.
+
+As a historical note, discussion of this RFC shed light on the powers of the minter to create *free outputs* in the coinbase transaction, which can be fixed independently of this proposal by limiting the powers of the coinbase transaction.
```

Comment:
This coinbase problem has nothing to do with this RFC. I don't think it's worth mentioning.